generators/app/index.js

```diff
@@ -428,7 +428,10 @@ module.exports = class extends BaseGenerator {
                     this.isGitInstalled(code => {
                         if (code === 0) {
                             this.gitExec('rev-parse --is-inside-work-tree', { trace: false }, (err, gitDir) => {
-                                if (!gitDir) {
+                                // gitDir has a line break to remove (at least on windows)
+                                if(gitDir && gitDir.trim() === 'true') {
```

Comment:
is the value normally a text `'true'`?

Reply:
When not in a git repo
```
> git rev-parse --is-inside-work-tree
fatal: Not a git repository (or any of the parent directories): .git
```
when in a git repo
```
> git rev-parse --is-inside-work-tree
true
```